wagtail/contrib/redirects/views.py

```diff
@@ -126,3 +133,183 @@ def add(request):
     return TemplateResponse(request, "wagtailredirects/add.html", {
         'form': form,
     })
+
+
+@permission_checker.require_any("add")
+def start_import(request):
+    from_encoding = "utf-8"
+    query_string = request.GET.get('q', "")
+
+    if not request.POST:
+        SUPPORTED_FORMATS = ["CSV", "TSV", "XLS", "XLSX"]
+        accepted_formats = [
+            x for x in DEFAULT_FORMATS if x.__name__ in SUPPORTED_FORMATS
+        ]
+        return render(
+            request,
+            "wagtailredirects/choose_import_file.html",
+            {
+                'search_form': SearchForm(
+                    data=dict(q=query_string) if query_string else None, placeholder=_("Search redirects")
+                ),
+                "form": ImportForm(accepted_formats),
+            },
+        )
+
+    form_kwargs = {}
+    form = ImportForm(
+        DEFAULT_FORMATS, request.POST or None, request.FILES or None, **form_kwargs
```

Comment:
Similar here - having a central source of truth in this file of what formats will be supported vs which formats are 'default' could make more sense. 

Again, I may be missing something but at first glance it looks like we are getting the supported formats in separate ways throughout this view.